journalbeat/cmd/instance/metrics.go

```diff
@@ -22,7 +22,7 @@ package instance
 import (
 	"fmt"
 
-	"github.com/coreos/go-systemd/sdjournal"
+	"github.com/coreos/go-systemd/v22/sdjournal"
```

Comment:
Is `go-systemd` changing the import path on any release? It seems rather inconvenient.

Reply:
It is going to be inconvenient for us every time we update a major release with go modules. At least they are adopting the modules approach, not like a few of our dependencies. :)